pkg/metrics/constants.go

```diff
@@ -33,6 +33,7 @@ const (
 	// Reasons for CREATE/DELETE shared metrics
 	ProvisionedReason = "provisioned"
 	ExpiredReason     = "expired"
+	RepairedReason    = "repaired"
```

Comment:
Should this be "Unhealthy"? To match the naming that you are using for the more specific metric